python/lsst/ts/nightreport/database.py

```diff
@@ -0,0 +1,68 @@
+# This file is part of ts_nightreport.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["NightReportDatabase"]
+
+import asyncio
+
+import sqlalchemy as sa
+import sqlalchemy.engine
+import structlog
+from sqlalchemy.ext.asyncio import create_async_engine
+
+
+class NightReportDatabase:
+    """Connection to the night report database and tables creation.
+
+    Create report table if it doesn't exist.
+
+    Parameters
+    ----------
+    nightreport_table: sa.Table
```

Comment:
Please add a space before the colon : and enclose the type in back quotes here and in line 41.